systemd/openqa-dehydrated.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Renewal openQA certificates.
+After=network.target nginx.service
+Requires=network.target
+Wants=openqa-setup-db.service
+
+[Service]
+Type=oneshot
+ExecStart= /usr/bin/dehydrated --cron | /opt/os-autoinst-scripts/filter-dehydrated-cron-output
```

Comment:
Are you sure this will work with the leading space?

Reply:
it should be a leading space. I will fix it (even if it is working🙏)